README.md

```diff
@@ -50,7 +50,16 @@ Here is how ThirdStats looks like on the Thunderbird default dark theme and ligh
 
 ## Privacy and Security
 
-ThirdStats is fully contained and doesn't contact any third-party CDN servers. It runs locally without any webserver. ThirdStats stores processed data in a cache, but you can clear and disable it. ThirdStats will never store this data elsewhere, nor sent or sell it anywhere. ThirdStats needs permissions to access accounts, read email headers and download export files. ThirdStats will always seek to minimise use of permissions where possible, and use granular permissions where available. Thunderbirds Permissions API is currently limited to access the whole email instead of the email header only.
+ThirdStats runs entirely locally and will never contact, send or sell data to, any third-parties. 
+
+ThirdStats features require [Thunderbird permissions](https://developer.thunderbird.net/add-ons/mailextensions/supported-webextension-api) to function, but will always seeks to minimise permissions where possible, and use granular permissions where available [1]:
```

Comment:
Both are correct usages. I'm Australian, so wrote 'minimise' (unconsciously).

 * Australian/British: `Minimise`
 * American: `Minimize`

Choices:

 1. Pick the LANG with which all **other documentation is already written and is consistent** (presumably your lang)
 2. Pick the LANG that is **most appropriate for the majority audiences** (America? Rest of world?)
 3. Roll a dice.

Reply:
I see thanks! I'm bad at languages, maybe someone has time bring some consistency here, but for me (as I'm not that good in languages in general) it would be sufficient to have a correct English word at all 😅 So I'll just keep it the way you proposed 👍🏻

Reply:
Your English is perfectly fine my friend :)